src/imars3dv2/filters/tilt.py

```diff
@@ -9,6 +9,8 @@
 from scipy.optimize import OptimizeResult
 from skimage.transform import rotate
 from skimage.registration import phase_cross_correlation
+from multiprocessing.managers import SharedMemoryManager
+from tqdm.contrib.concurrent import process_map
```

Comment:
I think you can remove `import concurrent.futures as cf` since it is no longer explicitly called.